Makefile

```diff
@@ -23,9 +23,14 @@ mixtral-tasks: src/mixtral-tasks.cpp
 	$(CXX) $(CXXFLAGS) -c src/mixtral-tasks.cpp -o mixtral-tasks.o
 tokenizer: src/tokenizer.cpp
 	$(CXX) $(CXXFLAGS) -c src/tokenizer.cpp -o tokenizer.o
+http: src/http.cpp
+	$(CXX) $(CXXFLAGS) -c src/http.cpp -o http.o
 
 main: src/main.cpp utils quants funcs socket transformer tasks llama2-tasks grok1-tasks mixtral-tasks tokenizer
 	$(CXX) $(CXXFLAGS) src/main.cpp -o main utils.o quants.o funcs.o socket.o transformer.o tasks.o llama2-tasks.o grok1-tasks.o mixtral-tasks.o tokenizer.o -lpthread
+
+server: src/server.cpp http utils quants funcs socket transformer tasks llama2-tasks grok1-tasks mixtral-tasks tokenizer
```

Comment:
Could we move this file to `src/apps/server.cpp`? Later I'll move `main.cpp` to `src/apps/main.cpp`. I would preffer to keep the framework in a single folder and applications in a different one.

Reply:
Definitely!